snapcraft/tests/plugins/test_kernel.py

```diff
@@ -1078,9 +1079,21 @@ def test_pull(self, download_mock):
         plugin.pull()
 
         download_mock.assert_called_once_with(
-            'core', 'stable', plugin.os_snap,
+            'core', self.options.kernel_channel, plugin.os_snap,
```

Comment:
Would you mind adding scenarios for all the channels?

Reply:
Just pushed an update for this.

Reply:
By `scenarios` I actually meant a list of values used to run the same test. Like what's done in KernelPluginDefaulTargetsTestCase. Sorry if that wasn't very clear.